File: semestre7/progLogFun/programas/pro.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Introduce un número: ");
        int n = int.Parse(Console.ReadLine());

        if (n % 2 == 0)
        {
            Console.WriteLine($"El número {n} es par.");
        }
        else
        {
            Console.WriteLine($"El número {n} es impar.");
        }

        for (int i = 0; i < n; i++)
        {
            Console.WriteLine("Yo");
        }
    }
}

File: semestre7/progLogFun/programas/pro.c
int main()
{
    printf("Introduce un numero:\n");
    int n;
    scanf("%d", &n);

    if (n % 2 == 0)
    {
        printf("El numero %d es par\n", n);
    }
    else
    {
        printf("El numero %d es impar\n", n);
    }

    for (int i = 0; i < n; i++)
    {
        printf("Yo\n");
    }

    return 0;
}